routemaster/server/tests/test_endpoints.py

```diff
@@ -326,6 +326,42 @@ def test_create_label_409_for_deleted_label(client, create_label):
     assert response.status_code == 409
 
 
+def test_restore_label(app, client, create_label, delete_label):
+    create_label('foo', 'test_machine', {})
```

Comment:
Just to test thoroughly, I'd put something in the metadata here so that we're testing that metadata isn't merged in the API, but actually replaced.

Reply:
d559606ffd22efd5ba8f0a45b7257ffb7f48a6b5